### PR TITLE
Restore correct ALQ dimension upon restart

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

**Comment by vkip** [comment]
jenkins build this please

**Comment by vkip** [comment]
jenkins build this please

File: opm/input/eclipse/Schedule/Well/Well.hpp [hunk 1]
```diff
@@ -369,6 +369,7 @@ class Well {
 
         void init_history(const DeckRecord& record);
         void init_vfp(const std::optional<VFPProdTable::ALQ_TYPE>& alq_type, const int vfp_table_nr, const UnitSystem& unit_system, const DeckRecord& record);
+        void updateALQDimension(const VFPProdTable& vfptable, const UnitSystem& usys);
```
Comment (bska) [inline opm/input/eclipse/Schedule/Well/Well.hpp:372]: This function (`Well::WellProductionProperties::updateALQWDimension()`) appears to be unimplemented, at least in the current PR.  Is that intentional?

**Review by bska — COMMENTED** [review]
This looks good for the most part.  It's a little bit annoying that the VFP tables aren't available in the restart file, but those have been the rules since forever so I'm not going to change that now.

I only spot one potential inconsistency here&ndash;a new function that is never implemented as far as I can tell&ndash;so maybe there's something missing?

File: opm/input/eclipse/Schedule/Well/Well.hpp (hunk 1 above)
Comment (vkip) [reply opm/input/eclipse/Schedule/Well/Well.hpp:372]: Eh, no - just forgot to delete from the first iteration, sorry about that. Will fix!

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**